### PR TITLE
Add products list

### PR DESCRIPTION
**Pull request by smacfarlane**
Initial pass at adding an endpoint for users to request products, optionally filtered by satellite, facility, sensor, processing level and date(-range).

Provides html, json, text and 'curl | bash' download script. 

No limits are currently placed on the number of items returned, so users could potentially be presented with a very very large list.

### CODE REVIEW

File: test/controllers/products_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ProductsControllerTest < ActionController::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (teknofire) [inline test/controllers/products_controller_test.rb]: Where are my tests?

Comment (smacfarlane) [inline test/controllers/products_controller_test.rb]: You haven't written them yet?

**Comment by smacfarlane** [comment]
@teknofire Ready for review

File: app/helpers/products_helper.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+module ProductsHelper
+  def collections_options klass, selected
+    logger.info(Array(selected).map(&:name))
```
Comment (teknofire) [inline app/helpers/products_helper.rb]: did you mean to leave in logger stuff?

File: app/views/products/_product.sh.haml [hunk 3]
```diff
@@ -0,0 +1 @@
+wget #{product.url('http://dds.gina.alaska.edu/nrt')} -nc -c
```
Comment (teknofire) [inline app/views/products/_product.sh.haml]: should this use the ENV variable that is setup for distribution?

**Comment by teknofire** [comment]
:+1:

**Pull request merged**